vue-frontend/src/config.js

```diff
@@ -6,7 +6,8 @@ export default Object.freeze({
   LINKEDIN_URL: "https://www.linkedin.com/company/canopasinc",
   MEDIUM_URL: "https://medium.com/canopas",
   BLOG_URL: "https://blog.canopas.com",
-  JOBS_URL: "https://jobs.canopas.com",
+  JOBS_URL: "https://dev-stack.canopas.com/jobs",
```

Comment:
I think we should not change this, instead we should use this link or router redirection based on the `IS_SHOW_JOBS` flag.